alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunnerService.kt

```diff
@@ -331,12 +331,12 @@ object MonitorRunnerService : JobRunner, CoroutineScope, AbstractLifecycleCompon
         when (job) {
             is Workflow -> {
                 launch {
-                    var lock: LockModel? = null
+                    var workflowLock: LockModel? = null
```

Comment:
change this to error log : https://github.com/opensearch-project/alerting/pull/1623/files#diff-7a7546a20937ab41ffd48d77f0d0ba9b9b6c5031289f2e2eb7543167b99701cbR236

Reply:
and catch between try and finally for lock acquire logic

Reply:
added catch between try & finally block.